probing/gap_preprocess.py

```diff
@@ -0,0 +1,137 @@
+import retokenize_edge_data as retokenize
```

Comment:
This shouldn't live in probing/, since it's a real target task.

Reply:
Context: Probing tasks are tasks on which don't reflect any intended application, and for which we generally train only a very small subset of the full parameters (so as to highlight what knowledge is already in the model). Neither of those conditions applies here.

Reply:
I'm doing another refactor that will move inputs for tasks that expect spans as input from probing to tasks, but that's worthy of a separate PR

Reply:
Sounds good.